### PR TITLE
fix losetup errors

### PR DESCRIPTION
**Pull request by 0pcom**
Fixes #

Changes:
- line 262 add sudo before losetup

Does this change need to mentioned in CHANGELOG.md?

Yes | No
yes

### CODE REVIEW

**Comment by 0pcom** [comment]
The build will fail at the next commit. This is merely due to the fact that it's now notifying you that `apt-get -y update` failed, as it should but wasn't doing so before.

There is also some initial repository configuration that is included in chroot-commands.sh. I can't test this until internet in the chroot is fixed

**Comment by 0pcom** [comment]
I've omitted my alternatiive method for setting the hypervisor key in the visors for the time being

Hotfixes for updating in the chroot without DNS.

I suggest as an alternative method to use caddy 2 server as a reverse proxy for the update mirrors - accessible via the localhost at specified ports.

the workaround did not work for the skyfleet.github.io/sky-update package repository, so these packages are fetched by the host's apt or with wget.

File: Makefile [hunk 1]
```diff
@@ -48,6 +59,142 @@ integration: build-skyconf ## runs integration tests.
 build-skyconf: ## builds skyconf.
 	${OPTS} go install ./cmd/skyconf
 
+skybian-skywire-package-amd64: #native build
```
Comment (evanlinjin) [inline Makefile:62]: I feel like we can put the majority of this logic here into scripts. This way, we wouldn't need so much code duplication.

File: build.sh [hunk 2]
```diff
@@ -129,46 +128,51 @@ create_folders()
...
-  mv "${PARTS_SKYWIRE_DIR}/bin/hypervisor" "${PARTS_SKYWIRE_DIR}/bin/skywire-hypervisor" || 0
-
-  info "Cleaning..."
-  rm -rf "${PARTS_SKYWIRE_DIR}/bin/README.md" "${PARTS_SKYWIRE_DIR}/bin/CHANGELOG.md"  || return 1
-
+#manually change the version when new packages are released - need automation for this!
+echo $(pwd)
+local _DST=${PARTS_SKYWIRE_DIR}/skywire-0.2.3-arm64.deb # Download destination file name.
+local _DST1=${PARTS_SKYWIRE_DIR}/skybian-skywire-0.1.2-arm64.deb
+local _DST2=${PARTS_SKYWIRE_DIR}/skybian-0.1.2-arm64.deb
```
Comment (evanlinjin) [inline build.sh:143]: Can we put the `.deb` names in the `build.conf` file instead?

File: build.sh [hunk 3]
```diff
@@ -129,46 +128,51 @@ create_folders()
...
+        notice "Creating skybian package from Makefile"
+        make skybian-package-arm64 || return 1
+        mv skybian-0.1.2-arm64.deb ${_DST2} || return 1
+      fi
+    else
+      #apt exists and is assumed working
+      sudo apt install -y software-properties-common
+      #configure the package repository on the host
+      sudo add-apt-repository 'deb http://skyfleet.github.io/sky-update sid main'
+      curl -L http://skyfleet.github.io/sky-update/KEY.asc | sudo apt-key add -
```
Comment (evanlinjin) [inline build.sh:164]: I'm sorry, we cannot add a community-maintained repository into our official build code.

**Review by evanlinjin — CHANGES_REQUESTED** [review]

File: build.sh (hunk 3 above)
Comment (0pcom) [reply build.sh:164]: I didn't think so, but I don't have access to the official ones so this was done for testing and demonstration purposes until an official repository containing built packages is created

Can you suggest an alternative or create the package repository officially?

File: Makefile (hunk 1 above)
Comment (0pcom) [reply Makefile:62]: yes

File: build.sh (hunk 2 above)
Comment (0pcom) [reply build.sh:143]: yes, probably

File: build.sh (hunk 3 above)
Comment (jdknives) [reply build.sh:164]: Depends on https://github.com/SkycoinProject/skywire-mainnet/issues/387

**Pull request closed without merging**